src/modules/bricks/volume-bricks/volume-bricks.js

```diff
@@ -300,6 +323,8 @@
 
         function _filterBrick(brick, filter) {
             var match = false,
+                percentage,
```

Comment:
Initialize the variable where they are used. if it never goes to last else if the percentage and utilization will become unused variables. 

var percentage = parseInt(filter.value);
var utilization = parseInt(brick.utilization.used);

Reply:
We can initialize it with 0 by default.

Reply:
@cloudbehl JavaScript has function scope, not block scope in ES5. It make sense to do it in ES6 as it has block scope. Tested it with jshint too. Didn't see any warning/error for it. 

@a2batic Initializing it with zero doesn't add much value to it as anyways its going to reinitialized with the entered utilization value. But, its good to have.